src/container/BooksList.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Book from '../components/Book';
+import { removeBook } from '../actions/index';
+import CategoryFilter from '../components/CategoryFilter';
+
+function BooksList() {
+  const allBooks = useSelector(state => state.book);
+  const filter = useSelector(state => state.filter);
+  const dispatch = useDispatch();
+  const handleRemoveBook = id => dispatch(removeBook(id));
+
+  return (
+    <div>
+      <h2>Books List</h2>
+      <table>
+        <thead>
+          <tr>
+            <th>Book ID</th>
+            <th>Title</th>
+            <th>Category</th>
+            <th> </th>
+          </tr>
+        </thead>
+        <tbody>
+          {allBooks
+            .filter(book => {
+              if (filter !== 'All') {
+                if (book.category === filter) {
+                  return true;
+                }
+                return false;
+              }
+              return true;
+            })
+            .map(book => (
+              <Book
+                key={book.id}
+                book={book}
+                handleRemoveBook={() => handleRemoveBook(book.id)}
+              />
+            ))}
+        </tbody>
+      </table>
+      <CategoryFilter />
```

Comment:
Great work guys!

I can see you are rendering `CategoryFilter ` component 👍 , but we also need to implement `handleFilterChange` and pass it down to `CategoryFilter`.

Screenshot from pathwright;
![image](https://user-images.githubusercontent.com/31897434/100337037-32b30800-2fd7-11eb-8b04-245b9f8f9cb6.png)